diff -wpruN '--exclude=*.orig' a~/hotspot/make/solaris/makefiles/sparcWorks.make a/hotspot/make/solaris/makefiles/sparcWorks.make
--- a~/hotspot/make/solaris/makefiles/sparcWorks.make	1970-01-01 00:00:00
+++ a/hotspot/make/solaris/makefiles/sparcWorks.make	1970-01-01 00:00:00
@@ -30,13 +30,13 @@ ifeq ($(SPEC),)
 
   # Note that this 'as' is an older version of the Sun Studio 'fbe', and will
   #   use the older style options. The 'fbe' options will match 'cc' and 'CC'.
-  AS	= /usr/ccs/bin/as
+  AS	= $(UTILS_CCS_BIN_PATH)as
 
-  NM    = /usr/ccs/bin/nm
+  NM    = $(UTILS_CCS_BIN_PATH)nm
   NAWK  = /bin/nawk
 
-  MCS	= /usr/ccs/bin/mcs
-  STRIP	= /usr/ccs/bin/strip
+  MCS	= $(UTILS_CCS_BIN_PATH)mcs
+  STRIP	= $(UTILS_CCS_BIN_PATH)strip
 endif
 
 REORDER_FLAG = -xF
